apps/yggdrasil/lib/yggdrasil/user.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Yggdrasil.User do
   use Ecto.Schema
   import Ecto.Changeset
+  import Ecto.Query, only: [from: 1, from: 2] # dislike this
```

Comment:
Haha no I didn't like querying for the default role I think. The import doesn't bother me. Don't remember why I put that there.